README.md

```diff
@@ -126,3 +174,22 @@ Standard](https://www.researchgate.net/publication/228971719_A_syntactic_specifi
 by Flor Narciso et al.  Many aspects of the grammar have been added to,
 modified, and in cases entirely rewritten to better support lark grammars and
 transformers.
+
+## Related, Similar, or Alternative Projects
+
+There are a number of similar, or related projects that are available.
+
+- ["MATIEC"](https://github.com/nucleron/matiec) - another IEC 61131-3 Structured
+Text parser which supports IEC 61131-3 second edition, without classes,
+namespaces and other fancy features. An updated version is also
+[available on Github](https://github.com/sm1820/matiec)
+- [OpenPLC Runtime Version 3](https://github.com/thiagoralves/OpenPLC_v3) -
+As stated by the project:
+  > OpenPLC is an open-source Programmable Logic Controller that is based on easy to use software. Our focus is to provide a low cost industrial solution for automation and research. OpenPLC has been used in many research papers as a framework for industrial cyber security research, given that it is the only controller to provide the entire source code.
+- [RuSTy](https://github.com/PLC-lang/rusty)
+[documentation](https://plc-lang.github.io/rusty/intro_1.html) - Structured text
+compiler written in Rust. As stated by the project:
+  > RuSTy is a structured text (ST) compiler written in Rust. RuSTy utilizes the LLVM framework to compile eventually to native code.
+- [IEC Checker](https://github.com/jubnzv/iec-checker) - Static analysis tool
+for IEC 61131-3 logic. As described by the maintainer:
+  > iec-checker has the ability to parse ST source code and dump AST and CFG to JSON format, so you can process it with your language of choice.
```

Comment:
Let's also add in TcBlack:
https://github.com/Roald87/TcBlack Python black-like code formatter for TwinCAT code

Reply:
Great! That's a good one... added!